BUILDING.md

```diff
@@ -1,152 +1,147 @@
 (including other Unix systems with X11.)
 
+#### CentOS / Fedora Linux
 
-### Ubuntu / Debian Linux
+>**Note** - On CentOS or an older version of Fedora, you may need to use `yum` instead of `dnf`.<br>
+>**Note 2** - On CentOS 7.x, you need to replace the `qwt-qt5-devel` package name with
+`qt5-qtbase-devel` in the `dnf install` line.<br>
+>**Note 3** - On CentOS 8 (Stream), you need to replace the `qt-devel` package name with `qt5-devel` in the `dnf install` line below.<br>
+>Make sure the `PowerTools` repo is enabled. For further information: https://access.redhat.com/discussions/5417621
 
 ```bash
-$ sudo apt install build-essential git cmake libsqlite3-dev qtchooser qt5-qmake qtbase5-dev-tools\
-    qttools5-dev-tools libsqlcipher-dev qtbase5-dev libqt5scintilla2-dev libqcustomplot-dev\
-    qttools5-dev
-$ git clone https://github.com/sqlitebrowser/sqlitebrowser
-$ cd sqlitebrowser
-$ mkdir build
-$ cd build
-$ cmake -Dsqlcipher=1 -Wno-dev ..
-$ make
-$ sudo make install
+sudo dnf install cmake gcc-c++ git qt-devel qt5-linguist qwt-qt5-devel sqlite-devel
+git clone https://github.com/sqlitebrowser/sqlitebrowser
+cd sqlitebrowser
+mkdir build && cd build
+cmake ..
+make
+sudo make install
 ```
 
-**Note** - Use `cmake -DFORCE_INTERNAL_QSCINTILLA=ON -Dsqlcipher=1 -Wno-dev ..` if you're using Debian and meet errors during compiling.
-
-This should complete without errors, giving you a binary file called 'sqlitebrowser'.
-
-Done. :)
-
-### CentOS / Fedora Linux
-
-**Note** - On CentOS or an older version of Fedora, you may need to use `yum` instead of `dnf`.
-
-**Note 2** - On CentOS 7.x, you need to replace the `qwt-qt5-devel` package name with
-`qt5-qtbase-devel` in the `dnf install` line below.
-
+This should complete without errors, and `sqlitebrowser` should now be launch-able from the command line.
 
-**Note 3** - On CentOS 8 (Stream), you need to replace the `qt-devel` package name with
-`qt5-devel` in the `dnf install` line below. Make sure the `PowerTools` repo is enabled.
+#### Debian / Ubuntu Linux
 
-```
-$ sudo dnf install cmake gcc-c++ git qt-devel qt5-linguist qwt-qt5-devel \
-       sqlite-devel
-$ git clone https://github.com/sqlitebrowser/sqlitebrowser
-$ cd sqlitebrowser
-$ cmake -Wno-dev .
-$ make
-$ sudo make install
+```bash
+sudo apt install build-essential cmake git libqcustomplot-dev libqt5scintilla2-dev libsqlcipher-dev \
+    libsqlite3-dev qt5-qmake qtbase5-dev qtbase5-dev-tools qtchooser qttools5-dev qttools5-dev-tools
+git clone https://github.com/sqlitebrowser/sqlitebrowser
+cd sqlitebrowser
+mkdir build && cd build
+cmake ..
+make
+sudo make install
 ```
 
-This should complete without errors, and `sqlitebrowser` should now be launch-able from the command line.
+>**Note** - Use `cmake -DFORCE_INTERNAL_QSCINTILLA=ON -Dsqlcipher=1 -Wno-dev ..`<br>
+>if you're using Debian and meet errors during compiling.
 
-### OpenSUSE
+This should complete without errors, giving you a binary file called `sqlitebrowser`. Done. :)
```

Comment:
Instead of "a binary file", *maybe* "an executable file"?

Not super sure, but it popped into my head as a potential alternative... :smile:

Reply:
Resolve via [fa3203f](https://github.com/sqlitebrowser/sqlitebrowser/pull/3485/commits/fa3203f39397851810d29ddbd061aab568132176)